packages/experimental/Icon/src/Icon.tsx

```diff
@@ -51,18 +51,13 @@ function renderFontIcon(iconProps: IconProps) {
 
 function renderSvg(iconProps: IconProps) {
   const svgIconProps: SvgIconProps = iconProps.svgSource;
-  const { width, height } = iconProps;
+  const { width, height, color } = iconProps;
```

Comment:
I'll need to revert my changes because I also fixed it but in a different way :)

Reply:
Ah sorry, I missed that 😞

Reply:
It's ok ;)